ARCHITECTURE.md

```diff
@@ -0,0 +1,194 @@
+# Standard Design and Architecture
+
+At the time of writing, almost a year of exploratory and freestyle project history has passed.
+Fortunately, it is not necessary for further understanding, so I'll spare you that.
+This document, though, lays out the design, architecture and direction of Standard.
+
+If the topics discussed herein are dear to you, please take it as an invitation to get involved.
+
+This design document shall be stable and amendments go through a proper process of consideration.
+
+## Overview
+
+Standard is a collection of functionality and best practices (_"framework"_) to bootstrap and sustain the automatable sections of the Software Delivery Lifecycle (SDLC) _efficiently_ with the power of Nix and Flakes.
```

Comment:
Many people might think "Software Development Lifecycle" when reading SDLC. Even though it is spelled out here it is easy to forget leading to a misunderstanding of the projects scope. Maybe we need a different acronym?